### DISCUSSION TITLE
24-hour time format

### DISCUSSION THREAD

**Discussion by gHashFlyer** [wojtekmaj/react-datetime-picker#176] [Q&A]
Is it possible for a user to select a 24-hour or military-based time such as 23:59 rather than 11:59 PM?

**Comment by wojtekmaj** [comment]
Yes. 

There are two important things to know:
* Locale is determined based on user's browser settings, and can be overwritten by setting `locale` prop.
* Time format is determined based on locale, and can also be overwritten by setting `format` prop.

So, depending on your use case, you could:
* Set `format` prop to use 24-hour time format
* Set `locale` prop to a locale that results in 24-hour time format

In general, I recommend setting `locale` to match page's language. I'd suggest setting `format` only when you need to be absolutely sure that time format is 24-hour no matter what.
